component/upgrade/patch-crds.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -eu
+
+for crd in ${CRDS_TO_PATCH}; do
+  exists=`kubectl get crd "${crd}" --ignore-not-found`
```

Comment:
Shellcheck complains here:

```
  exists=`kubectl get crd "${crd}" --ignore-not-found`
         ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
```

```suggestion
  exists=$(kubectl get crd "${crd}" --ignore-not-found)
```